build/distributions/CentOS/6/x86_64/DVD/6-simp_pkglist.txt

```diff
@@ -802,19 +805,19 @@ perl-IO-Compress-Zlib
 perl-List-MoreUtils
 perl-Module-Pluggable
 perl-Mozilla-LDAP
-perl-Net-Telnet
 perl-NetAddr-IP
+perl-Net-Telnet
 perl-Params-Validate
 perl-Pod-Escapes
 perl-Pod-Simple
-perl-Readonly
-perl-Readonly-XS
+# perl-Readonly
```

Comment:
Why did you comment out this and the subsequent package?  Were you able to build and ISO and run packer tests without this successfully?